dev/com.ibm.ws.microprofile.config.1.4/src/com/ibm/ws/microprofile/config14/impl/TimedCache.java

```diff
@@ -87,6 +87,10 @@ public TimedCache(ScheduledExecutorService executor, long delay, TimeUnit unit)
      */
     public V get(K key, Function<K, V> lookupFunction) {
 
+        if (delay == 0) {
```

Comment:
Please add a comment

Reply:
What happens if the system property is set to a negative value?? I don’t think it rounds up to 0? Maybe it should?? See here https://github.com/OpenLiberty/open-liberty/blob/67eeadf6e6ba211a338a1b3e518c1d031f256d72/dev/com.ibm.ws.microprofile.config.1.1/src/com/ibm/ws/microprofile/config/impl/AbstractConfigBuilder.java#L270